src/pages/workspace/categories/WorkspaceCategoriesPage.tsx

```diff
@@ -280,7 +282,19 @@ function WorkspaceCategoriesPage({policy, policyCategories, route}: WorkspaceCat
                         />
                         {isSmallScreenWidth && <View style={[styles.pl5, styles.pr5]}>{getHeaderButtons()}</View>}
                         <View style={[styles.ph5, styles.pb5, styles.pt3]}>
-                            <Text style={[styles.textNormal, styles.colorMuted]}>{translate('workspace.categories.subtitle')}</Text>
+                            {policy?.connections ? (
```

Comment:
```suggestion
                            {Object.keys(policy?.connections ?? {}).length ? (
```
connections could be an empty object

Reply:
I'm not sure that connections can be am empty object, but I agree that we can be cautious here 😄